Frontend/app/mentii.config.ts

```diff
@@ -4,12 +4,23 @@ export class MentiiConfig {
 
   getRootUrl():string {
     var url:string = 'http://api.mentii.me';
```

Comment:
When we first wrote this helper (before staging was even a thought) we decided that it would be more ideal that if for some reason this function failed, that the api would default to prod so that nothing breaks. Accidentally hitting prod from a dev branch is better than prod trying to hit 127.0.0.1